### PR TITLE
Ecf rubocop

### PR DESCRIPTION
**Pull request by ericcf**
@eschlange @michaelwehrley beware of this one because it touches so many files! Make sure you fetch latest master once it's merged!

### CODE REVIEW

File: app/controllers/think_feel_do_engine/coach/received_messages_controller.rb [hunk 1]
```diff
@@ -28,7 +29,7 @@ def reply_path(received_message)
         new_coach_group_message_path(
           @group,
           # This Message.class IS DeliveredMessage
-          body: "#{received_message.body}",
+          body: received_message.body,
```
Comment (michaelwehrley) [inline app/controllers/think_feel_do_engine/coach/received_messages_controller.rb]: nice catch.

File: app/models/emotion.rb [hunk 2]
```diff
@@ -17,12 +18,12 @@ def self.associate(participant, name)
     )
   end
 
-  private
-
   def self.normalized_name(n)
     n.strip.downcase
   end
 
+  private
```
Comment (michaelwehrley) [inline app/models/emotion.rb]: Why did you make this update?

File: app/models/wai_assessment.rb [hunk 3]
```diff
@@ -15,7 +18,7 @@ class WaiAssessment < ActiveRecord::Base
   private
 
   def scores_valid
-    return unless remove_nils.length > 0
+    return if remove_nils.empty?
```
Comment (michaelwehrley) [inline app/models/wai_assessment.rb]: Thank you.

File: app/models/emotion.rb (hunk 2 above)
Comment (ericcf) [inline app/models/emotion.rb]: `private` does not apply to class methods.

**Comment by michaelwehrley** [comment]
Everything looks great.  However I am not seeing the entire diff `Sorry, we could not display the entire diff because too many files (377) changed.`

File: app/models/emotion.rb (hunk 2 above)
Comment (michaelwehrley) [inline app/models/emotion.rb]: I had no idea.  👍

**Comment by michaelwehrley** [comment]
👍

**Comment by ericcf** [comment]
@eschlange you good if I merge this?

**Pull request merged**